cmd/bundle/plan.go

```diff
@@ -79,13 +80,59 @@ func newPlanCommand() *cobra.Command {
 			return root.ErrAlreadyPrinted
 		}
 
+		// Count actions by type and collect formatted actions
+		createCount := 0
+		updateCount := 0
+		deleteCount := 0
+		changed := make(map[string]bool)
+
+		for _, change := range plan.GetActions() {
+			changed[change.ResourceKey] = true
+			switch change.ActionType.StringShort() {
```

Comment:
Not blocking for this PR (cc @denik):

If new actions are added, they won't show up in the plan.

I recommend we use switch/case on the action type directly, and enumerate all action types.

Then, we can set up a linter to confirm that we cover all cases. When a new action type is added, it will fail unless the new type is added to the switch/case here.